internal/daemon/cluster/handlers/worker_service.go

```diff
@@ -92,12 +92,18 @@ func (ws *workerServiceServer) Status(ctx context.Context, req *pbs.StatusReques
 		})
 	}
 
+	// If not reported (in the case of an older worker), default to an active operational state
+	if wStat.OperationalState == "" {
+		wStat.OperationalState = server.ActiveOperationalState.String()
```

Comment:
I'm assuming if it's reporting status that it's active (my thought was it could be an older versioned worker that doesn't send operational state). Unknown state will still be sent work, so I'll update it to that.